package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "scripts": {
     "build": "./scripts/build",
-    "pretest": "eslint --ignore-path .eslintignore .",
+    "pretest": "eslint .",
```

Comment:
why ?

Reply:
Maybe it's from some older `eslint` version, but `.eslintignore` is now used as ignore file as default